packages/edge-gateway/wrangler.toml

```diff
@@ -108,8 +108,12 @@ workers_dev = true
 kv_namespaces = [{ binding = "DENYLIST" }]
 
 [env.test.vars]
+# TODO: setup smoke server for tests
+GOOGLE_CLOUD_API_URL = 'https://webrisk.googleapis.com'
+GOOGLE_CLOUD_API_KEY = 'TODO'
 IPFS_GATEWAYS = "[\"http://127.0.0.1:9081\", \"http://localhost:9082\", \"http://localhost:9083\"]"
 GATEWAY_HOSTNAME = 'ipfs.localhost:8787'
 EDGE_GATEWAY_API_URL = 'http://localhost:8787'
+CID_VERIFIER_URL = 'http://127.0.0.1:9111'
```

Comment:
This should be worker binding like we setup in https://github.com/web3-storage/reads/blob/edge-gateway-v1.0.4/packages/edge-gateway/wrangler.toml#L42-L46